src/applicationInsightsClient.ts

```diff
@@ -25,10 +29,15 @@ export class ApplicationInsightsClient {
                 "Connection String not found, please provide it before starting Application Insights SDK."
             );
         }
-        if (!this._config.getDisableStatsbeat()) {
-            this._statsbeat = new Statsbeat(this._config);
-            this._statsbeat.enable(true);
-        }
+        this._getStatsbeatInstrumentations();
```

Comment:
Let's move all Statsbeat logic to Statsbeat class, it would be easier to find like that